src/main/java/com/faforever/api/data/domain/User.java

```diff
@@ -19,4 +19,10 @@ public class User extends Login {
   public String getPassword() {
     return password;
   }
+
+  //Overriding toString method for efficient Logging
```

Comment:
Is there a particular reason not to use Lombok here?

Reply:
Yeah, If we use the ToString Annotation for User class then that annotation will have to call the super classes toString methods recursively as the User class inherit those properties from the classes up in the heirarchy, which leads to a messy format. Something like this User(super(super(id = 1232)), super(email = xyz@gmail.com)).

This is the reason why I used the traditional way of overriding the toString() method

Reply:
ok. But please make the fields lowercase :)

Reply:
@Brutus5000  I've added two new commits to this PR. Could you please check if the naming convention is alright now?